SayTheirNames/Source/Controller/Donations/DonationsMoreDetails/DonationsMoreDetailsController+Layout.swift

```diff
@@ -33,6 +33,35 @@ extension DonationsMoreDetailsController {
         let layout = UICollectionViewCompositionalLayout {
             (sectionIndex: Int,
             _ : NSCollectionLayoutEnvironment) -> NSCollectionLayoutSection in
+            
+            // Later Feature
```

Comment:
Is there a Trello card already to implement this? Might be a good idea to leave the TODO with the link so if people see it, they can check the card and implement it.

Reply:
The feature layout is already implemented, the reason to disable it is because there's no data support this section at this moment, it will yield this section always be empty.